frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -325,6 +333,10 @@ export const funnelLogic = kea<funnelLogicType>({
                 return false
             },
         ],
+        filtersDirty: [
```

Comment:
is this used to compare previous to current filters? kinda nit but `filtersDirty` confuses me a little 🤔 can we rename it to something more specific? Like `areNewFilters` or whatever if I'm understanding the selector correctly